fachhochschule-sudwestfalen.csl

```diff
@@ -1,24 +1,22 @@
-<?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" et-al-min="4" et-al-use-first="1" demote-non-dropping-particle="sort-only" default-locale="de-DE">
+﻿<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" et-al-use-first="1" et-al-subsequent-use-first="1" demote-non-dropping-particle="sort-only" default-locale="de-DE">
   <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
   <info>
-    <title>Fachhochschule Südwestfalen (Deutsch)</title>
```

Comment:
Please revert all.of these changes in info. The handling of Umlauts and other elements is standardized

Reply:
thanks for the head up, I'm pretty new to this. I made the changes to the original file, so things should look better in the recent pull request